apis/Google.Cloud.Firestore/Google.Cloud.Firestore/FirestoreConverterConfigurationAttribute.cs

```diff
@@ -0,0 +1,72 @@
+﻿// Copyright 2020, Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System;
+using System.Collections.Generic;
+
+namespace Google.Cloud.Firestore
+{
+    /// <summary>
+    /// Provides additional information for how Firestore converters (types implementing <see cref="IFirestoreConverter{T}"/>)
+    /// should be called.
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Method, AllowMultiple = true)]
```

Comment:
Why multiple?

Reply:
No particular reason, and definitely something to ask ourselves. I think it makes sense if it's inherited, in that a subclass may want to specify it, then a superclass add it later... if it's specified in multiple inheritance levels, I think it makes sense to allow it to be specified multiple times at one level too. But I think we should decide "method or class" for sure, then make those decisions (and experiment with overrides etc).